celery/backends/database/__init__.py

```diff
@@ -100,7 +100,7 @@ def ResultSession(self, session_manager=SessionManager()):
 
     @retry
     def _store_result(self, task_id, result, state,
-                      traceback=None, max_retries=3, **kwargs):
+                      args, worker, retries, queue, traceback=None, max_retries=3, **kwargs):
```

Comment:
If am not mistaken, these arguments are not passed to _store_result.
They are part of the `request` object that is passed as an argument.
I tried this code locally and it only worked when I used the request to retrieve this data.

Reply:
source: my source is https://github.com/celery/celery/blob/master/celery/backends/base.py#L701

Reply:
> 
> 
> source:https://github.com/celery/celery/blob/master/celery/backends/base.py#L701

I would happy accept any help with making this work. I am not familiar enough with the app to know what I am doing at this point. lol

Reply:
Resolved by @adimux